### PR TITLE
chore: Enables gofmt linting

### PR DESCRIPTION
**Pull request by gruyaume**
# Description

This PR enables the go fmt lint check and makes the necessary change to satisfy it.

### CODE REVIEW

**Comment by gab-arrobo** [comment]
ok to test

File: pcf_test.go [hunk 1]
```diff
@@ -38,9 +38,11 @@ func init() {
...
 }
 
-/*func init() {
-	factory.InitConfigFactory("amfTest/amfcfg.yaml")
-}*/
+/*
+	func init() {
+		factory.InitConfigFactory("amfTest/amfcfg.yaml")
+	}
+*/
```
Comment (gab-arrobo) [inline pcf_test.go:45]: Why not deleting this function instead of commenting it out?

File: pcf_test.go [hunk 2]
```diff
@@ -109,28 +111,30 @@ var Data = []byte(`{
...
+		time.Sleep(2 * time.Second)
+		if factory.AmfConfig.Configuration.SupportTAIList != nil &&
+			len(factory.AmfConfig.Configuration.SupportTAIList) == 2 {
+			fmt.Printf("test passed")
+		} else {
+			t.Errorf("test failed")
+		}
 	}
-}*/
+*/
```
Comment (gab-arrobo) [inline pcf_test.go:137]: Why not deleting this function instead of commenting it out?

File: pcf_test.go (hunk 1 above)
Comment (gruyaume) [reply pcf_test.go:45]: It was already commented out, here I simply ran `go fmt ./...` which formatted the comment correctly. We may want to completely delete it as well as many other unnecessary comments but I see this as a different topic.

File: pcf_test.go (hunk 2 above)
Comment (gruyaume) [reply pcf_test.go:137]: Please see my comment above

File: pcf_test.go (hunk 1 above)
Comment (gab-arrobo) [reply pcf_test.go:45]: ok

**Review by gab-arrobo — APPROVED** [review]
+1 (tested changes locally)

**Pull request merged**